addons/l10n_cl/data/uom_data.xml

```diff
@@ -10,32 +10,26 @@
 
       <record id="uom.product_uom_unit" model="uom.uom">
         <field name="l10n_cl_sii_code">10</field>
-        <field name="name">U</field>
```

Comment:
I made a fix in  `l10n_cl_edi` so that the values returned in the EDI are correct

Reply:
@jco-odoo this was made because of this:

https://www.aduana.cl/compendio-de-normas-anexo-51-b/aduana/2009-11-19/163937.html
https://www.aduana.cl/compendio-de-normas-anexo-51-b/aduana/2009-11-19/163937.html#vtxt_cuerpo_T26

Reply:
@Danisan Here is how I solved it

https://github.com/odoo/enterprise/pull/19547/commits/11577b766a8627c357d7557aeed38066af8199b2

Reply:
@william-andre thanks! good solution.